### PR TITLE
add example values for using Pyroscope

### PR DESCRIPTION
**Pull request by vankichi**
Signed-off-by: vankichi <kyukawa315@gmail.com>

### Description:

SSIA

### Environment:

- Go Version: 1.17.7
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.1

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by hlts2 — APPROVED** [review]
LGTM

File: example/helm/values-with-pyroscope.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        traefik.protocol: h2c
+        kubernetes.io/ingress.class: traefik
+      host: localhost
+      servicePort: grpc
+    gateway_config:
+      index_replica: 2
+
+agent:
+  # podAnnotations:
+  #   pyroscope.io/scrape: "true"
```
Comment (kevindiu) [inline example/helm/values-with-pyroscope.yaml:54]: why this part is commented?

Comment (vankichi) [reply example/helm/values-with-pyroscope.yaml:54]: I heard from @hlts2 that these annotations will be set when successful deployment.

**Review by hlts2 — DISMISSED** [review]
LGTM

File: example/helm/values-with-pyroscope.yaml (hunk 1 above)
Comment (kevindiu) [reply example/helm/values-with-pyroscope.yaml:55]: so should we remove these comment here?

File: example/helm/values-with-pyroscope.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
+defaults:
+  image:
+    tag: v1.4
+  time_zone: Asia/Tokyo
+  logging:
+    level: debug
+  server_config:
+    metrics:
+      pprof:
+        enabled: true
```
Comment (kevindiu) [inline example/helm/values-with-pyroscope.yaml:10]: I think it is better to let readers know which config enables Pyroscope, so I suggest:
```suggestion
        enabled: true # enable Pyroscope
```

File: example/helm/values-with-pyroscope.yaml (hunk 1 above)
Comment (vankichi) [reply example/helm/values-with-pyroscope.yaml:55]: we don't have to cuz we will mention it in blog posts or other documents.

Comment (vankichi) [reply example/helm/values-with-pyroscope.yaml:55]: @kevindiu 
https://github.com/vdaas/vald/pull/1582/files#diff-35aae708480cf8bc95340e5b5dd62d7902cbd1dcdecaf97e654ee413ce7e3418R53

**Review by kevindiu — APPROVED** [review]
LGTM

**Review by hlts2 — APPROVED** [review]
LGTM!!

**Pull request merged**